Frontend/src/pages/crowdactions/Detail/SocialMediaSharing.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Might be good to place this in the components directory, it's generally reusable

Reply:
Moved to higher level, and added to commit page as well. Noticed there already was a component. I replaced it with this one (has more options).